<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Расход материала</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Ткань</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3" rowstretch="0,5,2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>2</number>
        </property>
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="1" rowspan="2">
           <widget class="QDateEdit" name="de_material_from">
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="0" rowspan="2">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Дата с</string>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Расчитать</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3" rowspan="2">
           <widget class="QDateEdit" name="de_material_to">
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="5">
           <widget class="QPushButton" name="pushButton_3">
            <property name="text">
             <string>Подробно</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2" rowspan="2">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>По</string>
            </property>
           </widget>
          </item>
          <item row="0" column="6">
           <widget class="QPushButton" name="pushButton_4">
            <property name="text">
             <string>Печать</string>
            </property>
           </widget>
          </item>
          <item row="1" column="6">
           <widget class="QPushButton" name="pushButton_5">
            <property name="text">
             <string>Экспорт</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QTableWidget" name="tw_material">
          <property name="font">
           <font>
            <family>Tahoma</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>20</number>
          </attribute>
          <attribute name="verticalHeaderMinimumSectionSize">
           <number>18</number>
          </attribute>
          <column>
           <property name="text">
            <string>Ткань</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Приход кг</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Приход руб</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Расход кг</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Расход руб</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QTableWidget" name="tw_material_info">
          <property name="font">
           <font>
            <family>Tahoma</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>20</number>
          </attribute>
          <attribute name="verticalHeaderMinimumSectionSize">
           <number>18</number>
          </attribute>
          <column>
           <property name="text">
            <string>Название</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Расход кг</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Расход руб</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Фурнитура</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>2</number>
        </property>
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="2">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>По</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Дата с</string>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string>Расчитать</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDateEdit" name="dateEdit_3">
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QDateEdit" name="dateEdit_4">
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QTableWidget" name="tw_accessories"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ui_calc_material()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>471</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>558</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ui_calc_material_info()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>420</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>427</x>
     <y>-56</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ui_calc_material()</slot>
  <slot>ui_calc_material_info()</slot>
 </slots>
</ui>
